docs/gatsby-config.js

```diff
@@ -12,22 +12,16 @@ module.exports = {
         description: 'A guide to using rover',
         githubRepo: 'apollographql/rover',
         sidebarCategories: {
-          null: ['index'],
-          Contributing: [
-            'contributing/index',
-            'contributing/prerequisites',
-            'contributing/project-structure',
-            'contributing/documentation',
-          ],
-          Configuring: [
-            'usage/config/index',
-            'usage/config/authentication',
-            'usage/config/environment-variables',
-            'usage/config/profiles',
-            'usage/config/log'
-          ],
-          Privacy: [
-            `privacy/index`,
+          null: [
```

Comment:
no changes needed here, i'm just curious. why is this field called `null`?

Reply:
The key (like Privacy or `null` ) is the section heading. This is just an implementation detail of the gatsby theme. The `null` means that these items shoudn't be under a section heading